CHANGES.rst

```diff
@@ -221,7 +221,7 @@ Features
   `#4591 <https://pulp.plan.io/issues/4591>`_
 - Add ability to copy content between repositories, content type(s) can be specified.
   `#4716 <https://pulp.plan.io/issues/4716>`_
-- Renamed Errata/Update content to Advisory to better match the terminology of the RPM/DNF ecosystem.
+- Renamed Advisory/Update content to Advisory to better match the terminology of the RPM/DNF ecosystem.
   `#4902 <https://pulp.plan.io/issues/4902>`_
```

Comment:
I don't think we should change the wording of old changelogs.  They reflect the terminology of the issue/task they address.  We should focus on the rest of our documentation and doc strings.